packages/cli/src/ampx.ts

```diff
@@ -34,7 +34,21 @@ attachUnhandledExceptionListeners(usageDataEmitter);
 verifyCommandName();
 
 const parser = createMainParser(libraryVersion, usageDataEmitter);
-await parser.parseAsync(hideBin(process.argv));
+
+try {
+  await parser.parseAsync(hideBin(process.argv));
+} catch (e) {
```

Comment:
leave a comment here to explain that yargs invokes the command failure handler before rethrowing the error here. So we aren't actually swallowing the error, we're just preventing it from propagating to our unhandled exception handler and being printed again